workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerCreation/SendCreatePlayerRequestSystem.cs

```diff
@@ -51,8 +68,48 @@ private void RetryCreatePlayerRequest()
         protected override void OnUpdate()
         {
             if (PlayerLifecycleConfig.AutoRequestPlayerCreation && !initializationGroup.IsEmptyIgnoreFilter)
+            {
+                sendAutoPlayerCreationRequest = true;
```

Comment:
Why is this conditional on `!initializationGroup.IsEmptyIgnoreFilter` ?

Reply:
to prevent against the case where `AutoRequestPlayerCreation` is true but the worker hasn't connected yet

Reply:
That should never be the case - these systems are always created after the worker is created. Ideally they should be internal and only accessible through the helpers